test/CDB_OAuth.sql

```diff
@@ -36,14 +36,18 @@ CREATE TABLE test_tablesas AS SELECT * FROM test;
 CREATE VIEW test_view AS SELECT * FROM test;
 CREATE MATERIALIZED VIEW test_mview AS SELECT * FROM test;
 SELECT * INTO test_selectinto FROM test;
+CREATE FUNCTION test_function() RETURNS integer AS $$ BEGIN RETURN 1; END; $$ LANGUAGE PLPGSQL;
 
 SELECT * FROM test;
 SELECT * FROM test_tablesas;
 SELECT * FROM test_view;
 SELECT * FROM test_mview;
 SELECT * FROM test_selectinto;
+SELECT test_function();
+DROP FUNCTION test_function();
 
 \set QUIET on
+CREATE FUNCTION test_function() RETURNS integer AS $$ BEGIN RETURN 1; END; $$ LANGUAGE PLPGSQL;
```

Comment:
Do you need to drop and recreate the function here?

Reply:
Same with test2_function

Reply:
In order to cover the case in https://github.com/CartoDB/cartodb-postgresql/pull/379#issuecomment-550951903 and since we're testing that the owner can do what other users can't (like doing select for tables) we need to test that the drop can be done. In this step the action should be allowed and performed so we need to create the function again for the next check.